PowerEditor/src/NppCommands.cpp

```diff
@@ -999,6 +999,7 @@ void Notepad_plus::command(int id)
 			const auto current_index = _pDocTab->getCurrentTabIndex();
 			BufferID buffer_id = _pDocTab->getBufferByIndex(current_index);
 			_pDocTab->setIndividualTabColour(buffer_id, color_id);
+			_pDocTab->redraw();

```

Comment:
this should help with sync issue

Reply:
@ozone10 
Not really. Here's the way to reproduce (with your latest commit):
1. open 2 document 1.txt & 2.txt in the main view
2.  color 2.txt tab to green - we can see green color for 2.txt on both list & tab 
3. drag&drop 2.txt to sub view -  green color for 2.txt disappear on both list & tab (it's arguable if it's a bug, but...) 
4. drag&drop 2.txt back to main view -  green color is back on tab, but not on list (that's a bug for sure)

Reply:
Ok this should fix it, there was bug with tab color implementation for subview.

Reply:
Tested, fixed. Thank you!